phase1-wasm/Cargo.toml

```diff
@@ -12,8 +12,8 @@ edition = "2018"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-phase1 = { path = "../phase1", default-features = false }
-setup-utils = { path = "../setup-utils", default-features = false }
+phase1 = { path = "../phase1", default-features = true }
+setup-utils = { path = "../setup-utils", default-features = true }
```

Comment:
I don't think we can enable default features for the wasm crate, as it needs to be `no_std` - I think this is also why the CI is failing

Reply:
@ibaryshnikov @emmorais can you chime in here, please? We are not sure this is necessary. I can revert these specific changes but Round 1 completed with default-features enabled.